zypp/RepoInfo.cc

```diff
@@ -1047,6 +1048,26 @@ namespace zypp
     return str << "GpgCheck::UNKNOWN";
   }
 
+  bool RepoInfo::requireStatusWithMediaFile () const
+  {
+    bool canSkipMediaCheck = std::all_of( baseUrlsBegin(), baseUrlsEnd(), []( const zypp::Url &url ) { return url.schemeIsDownloading(); });
+
+    const auto &mDataPath = metadataPath();
+    if ( canSkipMediaCheck && !mDataPath.empty() ) {
+      zypp::Pathname mediafile = mDataPath/"media.1"/"media";
+
+      DBG << "Trying mediafile from: " << mediafile << std::endl;
```

Comment:
The line is IMO not needed. SUSEMediaVerifier logs an error if the file can't be parsed and the DBG at the end contains all info needed,

Reply:
OK